facia-press/app/frontpress/FapiFrontPress.scala

```diff
@@ -286,7 +286,7 @@ trait FapiFrontPress extends EmailFrontPress with GuLogging {
           "us" -> FaciaPressMetrics.UsPressLatencyMetric,
           "au" -> FaciaPressMetrics.AuPressLatencyMetric,
```

Comment:
Should we be supporting International and Europe here?!

https://github.com/guardian/frontend/pull/12429

https://eu-west-1.console.aws.amazon.com/cloudwatch/home?region=eu-west-1#metricsV2:graph=~(view~'timeSeries~stacked~false~metrics~(~(~'Application~'uk-press-latency~'ApplicationName~'facia-press~'Stage~'PROD))~region~'eu-west-1~start~'-PT168H~end~'P0D);query=~'*7bApplication*2cApplicationName*2cStage*7d*20uk-press-latency